tests/test_cli/test_ui_functions.py

```diff
@@ -0,0 +1,15 @@
+"""Tests for cli.ui.functions."""
+
+from unittest import mock
+
+from matcha_ml.cli.ui.functions import get_modify_resource_approval
+
+
+def test_get_modify_resource_approvals():
```

Comment:
This test current tests two different conditions, one where the user gives their approval and another where the user doesn't. A unit test should test a single, discrete condition - these should be split into two separate tests.

Reply:
Thanks Jon, they're split.